products/suse-manager.md

```diff
@@ -0,0 +1,77 @@
+---
+title: SUSE Manager
+category: server-app
+tags: suse
+iconSlug: suse
+permalink: /suse-manager
+versionCommand: zypper info SUSE-Manager-Server-release
+releasePolicyLink: https://www.suse.com/lifecycle
+changelogTemplate: "https://www.suse.com/releasenotes/x86_64/SUSE-MANAGER/{{'__RELEASE_CYCLE__'}}/"
+releaseColumn: true
+releaseDateColumn: true
+eolColumn: General Support
+
+releases:
```

Comment:
```suggestion
releases:
-   releaseCycle: "5.0"
    releaseDate: 2024-07-12
    eol: 2026-06-30
    latest: '5.0.0'
    latestReleaseDate: 2024-07-12
```

Based on https://www.suse.com/releasenotes/x86_64/SUSE-MANAGER/5.0/index.html.

Reply:
Good one, thanks for taking are of it!